src/components/Column/ColumnSettings.tsx

```diff
@@ -36,109 +41,89 @@ export const ColumnSettings: FC<ColumnSettingsProps> = ({id, name, color, visibl
-          >
-            <ArrowLeft />
-            {t("Column.addColumnLeft")}
-          </button>
-        </li>
-        <li>
-          <button
-            onClick={() => {
-              onClose?.();
-              handleAddColumn(index + 1);
-            }}
-          >
-            <ArrowRight />
-            {t("Column.addColumnRight")}
-          </button>
-        </li>
-        <li>
-          <button
-            onClick={() => {
-              onClose?.();
-              dispatch(Actions.deleteColumn(id));
-            }}
-          >
-            <Trash />
-            {t("Column.deleteColumn")}
-          </button>
-        </li>
-        <li>
-          <button
-            aria-label="Backlog Blue"
-            title="Backlog Blue"
-            className={classNames(getColorClassName("backlog-blue"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "backlog-blue", index, visible}))}
-          />
-          <button
-            aria-label="Planning Pink"
-            title="Planning Pink"
-            className={classNames(getColorClassName("planning-pink"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "planning-pink", index, visible}))}
-          />
-          <button
-            aria-label="Poker Purple"
-            title="Poker Purple"
-            className={classNames(getColorClassName("poker-purple"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "poker-purple", index, visible}))}
-          />
-          <button
-            aria-label="Value Violet"
-            title="Value Violet"
-            className={classNames(getColorClassName("value-violet"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "value-violet", index, visible}))}
-          />
-          <button
-            aria-label="Goal Green"
-            title="Goal Green"
-            className={classNames(getColorClassName("goal-green"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "goal-green", index, visible}))}
-          />
-          <button
-            aria-label="Yielding Yellow"
-            title="Yielding Yellow"
-            className={classNames(getColorClassName("yielding-yellow"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "yielding-yellow", index, visible}))}
-          />
-          <button
-            aria-label="Online Orange"
-            title="Online Orange"
-            className={classNames(getColorClassName("online-orange"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "online-orange", index, visible}))}
-          />
-        </li>
-      </ul>
+    <div ref={columnSettingsRef} className="column_settings">
+      <MiniMenu className="" items={menuItems} />
```

Comment:
I'd add a className, something like `column-settings__mini-menu`

Reply:
Just adding a classname to prevent it from being an empty string also seems unnecessary. If the classname wouldn't be used, just remove it by making the property optional.

Reply:
That was a relic tbh. I removed it, it's already an optional property